apex/transformer/pipeline_parallel/schedules/common.py

```diff
@@ -396,3 +397,11 @@ def backward_step(
 
     # timers("backward-compute").stop()
     return input_tensor_grad[0] if unwrap_input_tensor_grad else input_tensor_grad
+
+
+@contextmanager
```

Comment:
I understand this is moved from "fwd_bwd_no_pipelining.py", but would you know whether or not [`contextlib.nullcontext`](https://docs.python.org/3/library/contextlib.html#contextlib.nullcontext) suffices?

Reply:
I like that better. I'll make that change.